packages/widgets/src/tabbar.ts

```diff
@@ -127,6 +128,14 @@ class TabBar<T> extends Widget {
     return this._tabActivateRequested;
   }
 
+  /**
+   * A signal emitted when the tabbar add button is clicked.
+   *
+   */
+  get tabAddRequested(): ISignal<this, TabBar.ITabAddRequestedArgs> {
```

Comment:
I would have been inclined to use this signature `ISignal<this, void>`; did you have a purpose in mind for `TabBar.ITabAddRequestedArgs`?

Reply:
I think both here and below, I'd advocate for calling all these signals `addRequested`.

Reply:
Nope, didn't realize void could be used there.